docs/extend/extend-desktop/mvd-apptoappcommunication.md

```diff
@@ -82,6 +82,32 @@ export enum ActionType {       // not all actions are meaningful for all target
 
 Actions can be created dynamically at runtime, or saved and loaded by the system at login.
 
+### Cross-launch via URL
+
+Another way the Zowe Application Framework invokes Actions is via JSON Query Parameters, formated as a URL. This feature enables users to bookmark a set of application-to-application communication actions (in the form of a URL) that will be executed when opening the webpage. Developers creating separate web apps can build a link that will open Zlux and do specific actions in Zlux apps, for example, opening a file in the Editor.
```

Comment:
I don't think any of this should say zlux. That's a codename and not what the user sees. 
Instead of
`Developers creating separate web apps can build a link that will open Zlux and do specific actions in Zlux apps, for example, opening a file in the Editor.`
Try
`Developers creating separate web apps can build a link that will open the Zowe Desktop and do specific actions in Apps, for example, opening a file in the Editor.`